JokeRandomizer/JokeRandomizer/CustomButtons/CustomLabel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomLabel.swift
+//  JokeRandomizer
+//
+//  Created by john.adrian.de.claro on 12/7/22.
+//
+
+import UIKit
+
+
+
+class CustomLabel: UILabel {
+    
```

Comment:
use a private enum as constants for static texts

Reply:
Create a AppConstant File